std/range/package.d

```diff
@@ -5217,7 +5217,7 @@ auto sequence(alias fun, State...)(State args)
 
 // iota
 /**
-   Construct a range of values that span the given starting and stopping
+   Creates a range of values that span the given starting and stopping
```

Comment:
"This guy allows you to construct"

But Occam's Razor tells me that creates is probably better.